filecache.go

```diff
@@ -187,10 +188,12 @@ func (c *FileCache) GetFileName(filename string) string {
 	hashedFilename := md5.Sum([]byte(filename))
 	hashedDir := fnv.New32().Sum([]byte(filename))
 
-	var extension string
+	// If we don't find an original file extension, we'll default to this one
+	extension := c.DefaultExtension
+
 	// Look in the last 5 characters for a . and extension
 	lastDot := strings.LastIndexByte(filename, '.')
-	if lastDot > len(filename)-5 {
```

Comment:
This also needed to be longer because we might have `.docx` for example. Not using Go's `filepath.Ext()` because we don't want to backtrack into the whole filename looking for the last dot.